seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -775,19 +775,19 @@
       "description": [
         "To test your learning you will create a solution \"from scratch\".  Place your code between the indicated lines and it will be tested against multiple test cases.",
         "The algorithm to convert from Celsius to Fahrenheit is the temperature in Celsius times 9/5, plus 32.",
-        "You are given a variable <code>Tc</code> representing a temperature in Celsius.  Create a variable <code>Tf</code> and apply the algorithm to assign it the corresponding temperature in Fahrenheit."
+        "You are given a variable <code>celsius</code> representing a temperature in Celsius.  Create a variable <code>fahrenheit</code> and apply the algorithm to assign it the corresponding temperature in Fahrenheit."
       ],
       "releasedOn": "January 1, 2016",
       "challengeSeed": [
-        "function convert(Tc) {",
+        "function convert(celsius) {",
         "  // Only change code below this line",
         "  ",
         "",
         "  // Only change code above this line",
-        "  if(typeof Tf !== 'undefined') {",
-        "  return Tf;",
+        "  if ( typeof fahrenheit !== 'undefined' ) {",
+        "  return fahrenheit;",
         "  } else {",
-        "    return \"Tf not defined\";",
+        "    return 'fahrenheit not defined';",
```

Comment:
@erictleung Are you running this locally and tested the changes? I'm not quite sure about the changes that would come from removing the "\" from line 790.